calico-cloud_versioned_docs/version-19-1/get-started/connect/install-helm.mdx

```diff
@@ -0,0 +1,55 @@
+---
+description: Steps to connect your cluster to Calico Cloud.
+title: Install using Helm
+---
+
+# Connect a cluster to Calico Cloud using Helm
+
+You can connect your cluster to Calico Cloud by generating a unique Helm command in the Calico Cloud Manager UI and running it on your cluster.
+With Helm installations you can also set optional installation parameters to enable certain features.
+
+## Prerequisites
+
+* You have an active Calico Cloud account. You can sign up for a 14-day free trial at [calicocloud.io](https://calicocloud.io).
+* You are signed in to the Calico Cloud Manager UI as a user with the **Owner**, **Admin**, or **DevOps** role.
+* You have at least one cluster that meets our [system requirements](requirements/system-requirements.mdx).
+* You have kubectl access to the cluster.
+* You have installed Helm 3.0 or later on your workstation.
+
+## Install Calico Cloud using Helm
+
+1. From the **Managed Clusters** page, click **Connect Cluster**.
+1. In the **Connect a Cluster** dialog, enter a **Cluster Name** and select a **Cluster Type**.
+1. (Optional) If you must install a specific older release, select the Calico Cloud version you want to install.
+   We always recommend the latest version, which is installed by default.
+1. Click **Advanced Options**, and then select **Install via helm**
+1. Click **Connect** to generate a unique Helm installation command. Copy the command.
+
+   ```bash title="Example of generated Helm installation command"
```

Comment:
@jaderhs Are there reasons why any specifics of this generated command shouldn't be included?